src/Pact/Types/Exp.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE FunctionalDependencies #-}
```

Comment:
Please revert, massive reformatting (not hlint) + low value/suppressed suggestions